robyn/__init__.py

```diff
@@ -218,133 +220,115 @@ def inner(handler):
+    def delete(self, endpoint: str):
         """
         The @app.delete decorator to add a route with DELETE method
 
         :param endpoint str: endpoint to server the route
         """
 
         def inner(handler):
-            return self._add_route(
-                HttpMethod.DELETE, endpoint, handler, auth_required=auth_required
-            )
+            return self._add_route(HttpMethod.DELETE, endpoint, handler)
 
         return inner
 
-    def patch(self, endpoint: str, auth_required: bool = False):
+    def patch(self, endpoint: str):
         """
         The @app.patch decorator to add a route with PATCH method
 
         :param endpoint [str]: [endpoint to server the route]
         """
 
         def inner(handler):
-            return self._add_route(
-                HttpMethod.PATCH, endpoint, handler, auth_required=auth_required
-            )
+            return self._add_route(HttpMethod.PATCH, endpoint, handler)
 
         return inner
 
-    def head(self, endpoint: str, auth_required: bool = False):
+    def head(self, endpoint: str):
         """
         The @app.head decorator to add a route with HEAD method
 
         :param endpoint str: endpoint to server the route
         """
 
         def inner(handler):
-            return self._add_route(
-                HttpMethod.HEAD, endpoint, handler, auth_required=auth_required
-            )
+            return self._add_route(HttpMethod.HEAD, endpoint, handler)
 
         return inner
 
-    def options(self, endpoint: str, auth_required: bool = False):
+    def options(self, endpoint: str):
         """
         The @app.options decorator to add a route with OPTIONS method
 
         :param endpoint str: endpoint to server the route
         """
 
         def inner(handler):
-            return self._add_route(
-                HttpMethod.OPTIONS, endpoint, handler, auth_required=auth_required
-            )
+            return self._add_route(HttpMethod.OPTIONS, endpoint, handler)
 
         return inner
 
-    def connect(self, endpoint: str, auth_required: bool = False):
+    def connect(self, endpoint: str):
         """
         The @app.connect decorator to add a route with CONNECT method
 
         :param endpoint str: endpoint to server the route
         """
 
         def inner(handler):
-            return self._add_route(
-                HttpMethod.CONNECT, endpoint, handler, auth_required=auth_required
-            )
+            return self._add_route(HttpMethod.CONNECT, endpoint, handler)
 
         return inner
 
-    def trace(self, endpoint: str, auth_required: bool = False):
+    def trace(self, endpoint: str):
         """
         The @app.trace decorator to add a route with TRACE method
 
         :param endpoint str: endpoint to server the route
         """
 
         def inner(handler):
-            return self._add_route(
-                HttpMethod.TRACE, endpoint, handler, auth_required=auth_required
```

Comment:
basically all the `auth_required` methods in this file.

Reply:
Yeah, our fork wasn't up to date when those changes were made. I'll fix that by the next commit